src/modules/commander/state_machine_helper.h

```diff
@@ -115,6 +115,12 @@ enum class arm_disarm_reason_t {
 	UNIT_TEST = 13
 };
 
+enum RCLossExceptionBits {
```

Comment:
Is `enum class` not possible?

Reply:
I'll check, I was honestly following existing patterns assuming it was already attempted there.

Reply:
Ah now I remember you'd have to implement the operators when you define your own class.
![image](https://user-images.githubusercontent.com/4668506/123113316-be422980-d43e-11eb-9160-bfee16531bd8.png)
see also https://stackoverflow.com/questions/32578638/how-to-use-c11-enum-class-for-flags
So it's very cumbersome for bitfield definitions if you don't have a specific bitfield type to inherit from already. Not sure how much value that would add.

Reply:
I see. Up to you.